### PR TITLE
Clarify S3 backup secret info, add docs links to address rancher/docs #3054

### PR DESCRIPTION
**Pull request by catherineluse**
https://github.com/rancher/docs/issues/3054

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,31 @@ It installs the following cluster-scoped CRDs:
...
+For help configuring the storage location, see the [documentation.](https://rancher.com/docs/rancher/v2.5/en/backups/configuration/storage-config/)
+
+---
+
+### S3 Credentials
+
+If you are using S3 to store your backups, the Backup custom resource can reference an S3 credential secret in any namespace. The `credentialSecretNamespace` directive tells the backup application where to look for the secret:
+
+```
+
```
Comment (jiaqiluo) [inline README.md:79]: nit: remove the empty line. 
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -64,6 +64,31 @@ It installs the following cluster-scoped CRDs:
 2. Performing a backup: To take a backup, user has to create an instance of the Backup CRD (create a Backup CR). Each Backup CR must reference a ResourceSet. A Backup CR can be used to perform a one-time backup or recurring backups. Refer [examples](https://github.com/rancher/backup-restore-operator/tree/master/examples) folder for sample manifests
 3. Restoring from a backup: To restore from a backup, user has to create an instance of the Restore CRD (create a Restore CR). A Restore CR must contain the exact Backup filename.  Refer [examples](https://github.com/rancher/backup-restore-operator/tree/master/examples) folder for sample manifests
 
+---
+### Storage Location
+
+For help configuring the storage location, see the [documentation.](https://rancher.com/docs/rancher/v2.5/en/backups/configuration/storage-config/)
```
Comment (jiaqiluo) [inline README.md:70]: nit.
```suggestion
For help configuring the storage location, see the [documentation].(https://rancher.com/docs/rancher/v2.5/en/backups/configuration/storage-config/)
```

File: README.md [hunk 3]
```diff
@@ -51,9 +51,9 @@ kubectl delete namespace cattle-resources-system
 
 It installs the following cluster-scoped CRDs:
 #### Backup
-  A backup can be performed by creating an instance of the Backup CRD. It can be configured to perform a one-time backup, or to schedule recurring backups.
+  A backup can be performed by creating an instance of the Backup CRD. It can be configured to perform a one-time backup, or to schedule recurring backups. For help configuring backups, see [this documentation.](https://rancher.com/docs/rancher/v2.5/en/backups/configuration/backup-config/)
 #### Restore
-  Creating an instance of the Restore CRD lets you restore from a backup file.
+  Creating an instance of the Restore CRD lets you restore from a backup file. For help configuring backups, see [this documentation.](https://rancher.com/docs/rancher/v2.5/en/backups/configuration/restore-config/)
```
Comment (jiaqiluo) [inline README.md:56]:
```suggestion
  Creating an instance of the Restore CRD lets you restore from a backup file. For help configuring restores, see [this documentation].(https://rancher.com/docs/rancher/v2.5/en/backups/configuration/restore-config/)
```

File: README.md [hunk 4]
```diff
@@ -64,6 +64,31 @@ It installs the following cluster-scoped CRDs:
...
+---
+### Storage Location
+
+For help configuring the storage location, see the [documentation.](https://rancher.com/docs/rancher/v2.5/en/backups/configuration/storage-config/)
+
+---
+
+### S3 Credentials
+
+If you are using S3 to store your backups, the Backup custom resource can reference an S3 credential secret in any namespace. The `credentialSecretNamespace` directive tells the backup application where to look for the secret:
```
Comment (jiaqiluo) [inline README.md:76]:
```suggestion
If you are using S3 to store your backups, the `Backup` custom resource can reference an S3 credential secret in any namespace. The `credentialSecretNamespace` directive tells the backup application where to look for the secret:
```

**Review by nickgerace — APPROVED** [review]
LGTM with the aforementioned suggestions.

**Review by ryansann — APPROVED** [review]

**Review by jiaqiluo — COMMENTED** [review]
LGTM

**Pull request merged**